Add Nidalee Q

lzardy opened the pull request:
Makes use of the AddProjectileCoords function and is the first script to have a damage ramp-up effect.

lzardy committed:
Add Nidalee Q

Makes use of the AddProjectileCoords function and is the first script to have a damage ramp-up effect.

FurkanS1821 reviewed (approved):
A lot of maths I can't understand but if it is tested, then it is okay.

danil179 commented:
Requires LeagueSandbox/GameServer#827

lzardy committed:
Adapt rename of AddProjectileCoords

AddProjectileCoords was renamed to AddProjectile, so this changes all uses of AddProjectile to use AddProjectileCoords' parameters.
